frontend/src/Indexer/Add/AddIndexerModalContent.css

```diff
@@ -28,3 +28,42 @@
 .scroller {
   flex: 1 1 auto;
 }
+
+.filterRow {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.filterContainer {
+  display: flex;
+  align-items: stretch;
+  flex: 1;
+  flex-direction: column;
+  margin-right: 12px;
+}
+
+.filterContainer:last-child {
+  margin-right: 0;
+}
+
+.filterLabel {
+  margin-bottom: 3px;
+  font-weight: bold;
+}
+
+@media only screen and (max-width: $breakpointSmall) {
+  .filterRow {
+    flex-direction: column;
+  }
+
+  .filterContainer {
+    margin-right: 0;
+    margin-bottom: 12px;
+  }
+
+  .scroller {
+    margin-right: -30px;
+    margin-bottom: -30px;
+    margin-left: -30px;
+  }
+}
```

Comment:
Add mobile override for `alert` class with `display: none` as discussed. Then good.

Reply:
Ok, just added it in my most recent push